### PR TITLE
Chopper units bug

### LINKED ISSUE #21: Chexus looks for rotation speed units in the wrong place
> The problem is that chexus looks for a units attr on the rotation_speed group: https://github.com/scipp/chexus/blob/b14576ef026c24b41e4436bd59225ff2dc68d474/src/chexus/validators.py#L246
But the file only contains units on the value group.

### PR DESCRIPTION
**Pull request by jokasimr**
Fixes #21 #20

### CODE REVIEW

File: src/chexus/validators.py [hunk 1]
```diff
@@ -243,14 +243,16 @@ def applies_to(self, node: Dataset | Group) -> bool:
...
 
-        if 'units' in node.children.get('rotation_speed').attrs:
-            unit = node.children.get('rotation_speed').attrs.get('units')
-            try:
-                sc.scalar(1, unit=unit).to(unit='Hz')
-            except sc.UnitError:
-                pass
-            else:
-                return
+        if 'NXlog' == node.children.get('rotation_speed').attrs.get('NX_class'):
```
Comment (SimonHeybrock) [inline src/chexus/validators.py:246]: What if `rotation_speed` is not a `Group`? Could it be a simple `Dataset` if not time dependent?

File: src/chexus/validators.py [hunk 2]
```diff
@@ -243,14 +243,16 @@ def applies_to(self, node: Dataset | Group) -> bool:
...
+            log = node.children.get('rotation_speed')
```
Comment (SimonHeybrock) [inline src/chexus/validators.py:247]: Maybe you can simply handle both cases, without the testing, something like

```python
rotation_speed = node.children.get('rotation_speed')
if rotation_speed.attrs.get('NX_class') == 'NXlog':
  value = rotation_speed.children.get('value')
else:
  value = rotation_speed
# now check units
```

File: tests/validators_test.py [hunk 3]
```diff
@@ -325,12 +325,16 @@ def test_NXdisk_chopper_units(units: str, good: bool):
         name='x',
         attrs={'NX_class': 'NXdisk_chopper'},
     )
-    group.children['rotation_speed'] = chexus.Dataset(
+    group.children['rotation_speed'] = chexus.Group(
         name='x/rotation_speed',
+        attrs={'NX_class': 'NXlog'},
+    )
+    group.children['rotation_speed'].children['value'] = chexus.Dataset(
```
Comment (SimonHeybrock) [inline tests/validators_test.py:332]: Keep old test and add a new one with NXlog?

File: src/chexus/validators.py (hunk 1 above)
Comment (jokasimr) [reply src/chexus/validators.py:246]: Yes it could be, but not at ess. Do you prefer it to be able to handle both cases?

Comment (SimonHeybrock) [reply src/chexus/validators.py]: Yes.

**Pull request merged**